app/src/main/java/com/parking_system_kotlin/utils/DateUtils.kt

```diff
@@ -0,0 +1,35 @@
+package com.parking_system_kotlin.utils
+
+import java.text.ParseException
+import java.text.SimpleDateFormat
+import java.util.*
+
+open class DateUtils {
```

Comment:
why is this class open?

Reply:
it was like that because i used it in the tests, but im not longer using it so, changed the open class type